src/pages/api/buildRecList.js

```diff
@@ -0,0 +1,15 @@
+import { useBuildRecListService } from "./services/buildRecList.service";
+
+export default async function buildRecs(req, res) {
+  const responseId = req.body;
+  console.log({ responseId });
+  //const { recList } = useBuildRecListService({ responseId });
+  const { buildGameRecList } = useBuildRecListService( {responseId} );
```

Comment:
![image](https://github.com/dotGrimmz/royal-recommendations/assets/47502785/189dc8b8-0116-4f8e-8672-f37014a7bf74)

This should should give you a hint on why its showing up undefined in your service file.

Reply:
This one has been resolved! Didn't know the param names had to be the same